docs/list_of_theory_docs.jl

```diff
@@ -10,6 +10,7 @@ theory_docs = Any[
     "Atmos" => Any[
         "AtmosModel" => "Theory/Atmos/AtmosModel.md",
         "EDMF Model" => "Theory/Atmos/EDMF_plots.md",
+        "Aerosol-activation" => "Theory/Atmos/Aerosol-activation.md",
```

Comment:
I would suggest changing the title and file name to follow the style in other files. We should then also change the name of the markdown file.
```suggestion
        "Aerosol Activation" => "Theory/Atmos/Aerosol_activation.md",
```